test/support/utils.rb

```diff
@@ -178,7 +178,7 @@ def prepare_test_environment(program, test_steps, &block)
     TIMEOUT_SEC = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 10).to_i
 
     def run_test_scenario cmd, test_info
-      PTY.spawn(cmd) do |read, write, pid|
+      PTY.spawn({ "HOME" => @pty_home_dir }, cmd) do |read, write, pid|
```

Comment:
@ko1 I think this is better than updating `ENV["HOME"]`. Wdyt?

Reply:
It seems good idea but Dir.tmpdir is not removed automatically.

Reply:
Good catch. I've updated it.

Reply:
It seems too much to make a directory per tesst. `at_exit` will help you.

Reply:
Yeah. I eventually picked the [startup/shutdown callbacks](https://www.rubydoc.info/gems/test-unit/Test/Unit/TestCase) instead.